net_connection/src/protocol_wrapper.rs

```diff
@@ -34,13 +34,13 @@ pub struct ConfigData {
 
 #[derive(Serialize, Deserialize, Debug, Clone, PartialEq, DefaultJson)]
 pub struct ConnectData {
-    pub address: String,
+    pub address: Address,
 }
 
 #[derive(Serialize, Deserialize, Debug, Clone, PartialEq, DefaultJson)]
 pub struct PeerData {
```

Comment:
@thedavidmeister @neonphog perhaps a change on this side would require a change on the n3h side?

Reply:
@Connoropolous i just chatted to @neonphog about these changes in teal, so he's aware, all good for merge i think :)